bamboo/bootstrap-publish.sh

```diff
@@ -3,11 +3,14 @@ set -e
 . ./bamboo/set-bamboo-env-variables.sh
 . ./bamboo/abort-if-not-publish.sh
 
+# Extract cache of compiled TS files
+./bamboo/extract-ts-build-cache.sh
+
 npm install
 rm -rf website/build
 npm run docs-install
 npm run docs-build
-npm run bootstrap-no-build
+npm run bootstrap-no-build-ci
```

Comment:
nit:   This should probably be bootstrap-<something>-no-tsc, but at some point we're being absurd.   

Not your PR, I just really don't like that entire bootstrap* block at this point for convention reasons.    It should probably be namespaced or something.

Reply:
Since I'm adding these scripts, I'm fine to namespace them. What are we thinking? `ci:bootstrap-no-tsc`?

Reply:
ci:<scripts for ci> .... yes, that works.

Reply:
a58c5b0b78be0d68b79c400a735a5744a921b42e